packages/webdriverio/src/utils/index.js

```diff
@@ -510,3 +510,36 @@ export const updateCapabilities = async (params, automationProtocol) => {
         }
     }
 }
+
+/**
+ * compare if an object (`base`) contains the same values as another object (`match`)
+ * @param {object} base  object to compare to
+ * @param {object} match object that needs to match thebase
+ */
+export const containsObject = (base, match) => {
+    for (const [key, value] of Object.entries(match)) {
+        if (typeof base[key] === 'undefined' || base[key] !== value) {
+            return false
+        }
+    }
+
+    return true
+}
+
+/**
+ * Helper utility to check file access
+ * @param {String} file file to check access for
+ * @return              true if file can be accessed
+ */
+export const canAccess = (file) => {
+    if (!file) {
+        return false
+    }
+
+    try {
+        fs.accessSync(file)
```

Comment:
Can we avoid using sync functions in Promise?

canAccess is called in async function above

Reply:
What is the harm of using the synchronous version here? We use it for other `fs` functions to and there is no significant burden to it.

Reply:
Ok, let's do this way